ui/admin/app/routes/scopes/scope/roles/role/manage-scopes/manage-custom-scopes.js

```diff
@@ -7,7 +7,7 @@ import Route from '@ember/routing/route';
 import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
 import { TYPE_SCOPE_PROJECT } from 'api/models/scope';
-
+import { TrackedObject } from 'tracked-built-ins';
```

Comment:
small nitpick: Could we add a space between the imports and class declaration?

Reply:
of course!